### ISSUE TITLE
Improve location selection

### ISSUE THREAD

**Issue by glpzzz** [cuba-weather/cuba-weather-flutter#18]
You can use 2 drop-downs. One for province and other for municipality. 

Or a tree like 

Or a drop-down with search. 

Or perhaps remember the last consulted location

**Comment by leynier** [comment]
@glpzzz Currently the app uses an auto completer edit text

**Comment by leynier** [comment]
![photo_2020-01-19_01-33-05](https://user-images.githubusercontent.com/36774373/72676145-e2d0ef80-3a5b-11ea-9e48-ee6a376ea78e.jpg)

**Comment by leynier** [comment]
https://drive.google.com/file/d/1XrXPuvHKvgVHyClU3c9x2FZqrCGzWAL6/view?usp=sharing

**Comment by glpzzz** [comment]
It seems I don't have the last one. I downloaded from apklis

On January 19, 2020 1:28:24 AM GMT-05:00, "Leynier Gutiérrez González" <notifications@github.com> wrote:
>@glpzzz Currently the app uses an auto completer edit text
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/leynier/cuba-weather-flutter/issues/18#issuecomment-575971974

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by leynier** [comment]
Yes, sorry, is that until Monday I can not upload updates of the application for issues related to Apklis.

> It seems I don't have the last one. I downloaded from apklis
> […](#)
> On January 19, 2020 1:28:24 AM GMT-05:00, "Leynier Gutiérrez González" ***@***.***> wrote: @glpzzz Currently the app uses an auto completer edit text -- You are receiving this because you were mentioned. Reply to this email directly or view it on GitHub: [#18 (comment)](https://github.com/leynier/cuba-weather-flutter/issues/18#issuecomment-575971974)
> -- Sent from my Android device with K-9 Mail. Please excuse my brevity.
